cmake/Doxyfile.in

```diff
@@ -228,73 +254,40 @@ TAB_SIZE               = 8
-#    **/
-
-ALIASES  += trace{4}="@addtogroup trace_marker_\1\n @par\n - { \"\2\":\"\3\" } <br> \4"
-ALIASES  += trace{6}="@addtogroup trace_marker_\1\n @par\n - { \"\2\":\"\3\", \"\4\":\"\5\" } <br> \6"
-ALIASES  += trace{8}="@addtogroup trace_marker_\1\n @par\n - { \"\2\":\"\3\", \"\4\":\"\5\", \"\6\":\"\7\" } <br> \8"
-ALIASES  += trace{10}="@addtogroup trace_marker_\1\n @par\n - { \"\2\":\"\3\", \"\4\":\"\5\", \"\6\":\"\7\", \"\8\":\"\9\" } <br> \10"
-ALIASES  += trace{12}="@addtogroup trace_marker_\1\n @par\n - { \"\2\":\"\3\", \"\4\":\"\5\", \"\6\":\"\7\", \"\8\":\"\9\", \"\10\":\"\11\" } <br> \12"
-ALIASES  += interfaceTraceStarted{4}="@addtogroup trace_marker_\1\n @par\n - { \"InterfaceName\":\"\2\", \"Action\":\"\3\", \"Status\":\"Started\" } <br> \4"
-ALIASES  += interfaceTraceSucceeded{4}="@addtogroup trace_marker_\1\n @par\n - { \"InterfaceName\":\"\2\", \"Action\":\"\3\", \"Status\":\"Succeeded\" } <br> \4"
-ALIASES  += interfaceTraceFailed{4}="@addtogroup trace_marker_\1\n @par\n - { \"InterfaceName\":\"\2\", \"Action\":\"\3\", \"Status\":\"Failed\" } <br> \4"
-ALIASES  += interfaceTraceStopped{4}="@addtogroup trace_marker_\1\n @par\n - { \"InterfaceName\":\"\2\", \"Action\":\"\3\", \"Status\":\"Stopped\" } <br> \4"
-
-ALIASES  += "project=\par Project:<br>" \
-            "global=\xrefitem global \"Global\" \"Globals\" [rw]" \
-            "global_r=\xrefitem global \"Global\" \"Globals\" [r]" \
-            "global_w=\xrefitem global \"Global\" \"Globals\" [w]" \
-            "global_rw=\xrefitem global \"Global\" \"Globals\" [rw]" \
-            "global{1}=\xrefitem global \"Global\" \"Globals\" [\1]" \
-            "serviceid=\xrefitem serviceid \"Service ID\" \"Service IDs\"" \
-            "synchronism=\par Synchronism:<br>" \
-            "reentrancy=\par Reentrancy:<br>" \
-            "swsid=\xrefitem swsid \"SWS Id\" \"SWS Ids\"" \
-            "tcswsid=\xrefitem tcswsid \"Tested SWS Id\" \"Tested SWS Ids\"" \
-            "lhref=\xrefitem lhref \"Referenced Lastenheft Id\" \"Referenced Lastenheft Ids\"" \
-            "tclhref=\xrefitem tclhref \"Tested Lastenheft Id\" \"Tested Lastenheft Ids\"" \
-            "swsref{1}=\ref swsid \"\1\"" \
-            "texref{1}=\ref tex \"\1\"" \
-            "tcinput=\par Input:<br>" \
-            "tctc=\par Testcase:<br>" \
-            "tcoutput=\par Expected Output:<br>"
+# newlines (in the resulting output). You can put ^^ in the value part of an
+# alias to insert a newline as if a physical newline was in the original file.
+# When you need a literal { or } or , in the value part of an alias you have to
+# escape them by means of a backslash (\), this can lead to conflicts with the
+# commands \{ and \} for these it is advised to use the version @{ and @} or use
+# a double escape (\\{ and \\})
+
+ALIASES                = "trace{4}=@addtogroup trace_marker_\1\n @par\n - { \"\2\":\"\3\" } <br> \4" \
```

Comment:
I think those may have been copied from an unrelated project. Can you check whether we ever use any of those in our project, and if not, just drop those aliases?